CardForm/src/main/java/com/braintreepayments/cardform/view/CvvEditText.java

```diff
@@ -61,6 +64,19 @@ public void setCardType(CardType cardType) {
         invalidate();
     }
 
+    /**
+     * @param masked if {@code true}, this field will be masked.
+     */
+    public void setMasked(boolean masked) {
```

Comment:
Should this logic match `setMasked` for `CardEditText`? If so, should we move it to `ErrorEditText`?

Reply:
They're slightly different. We want to match the functionality on iOS where the card number has a custom mask only when you switch focus, while the CVV we want to mask as an ordinary password.

Reply:
Why not just change the input type to password here? Is the goal to have more consistency using transformations?